tests/acceptance/README.md

```diff
@@ -5,34 +5,11 @@ Acceptance tests for Mender
 Prerequisites
 -------------
 
-* PyTest
-* [Fabric](http://www.fabfile.org)
+* OS dependencies, assuming Debian based, at `requirements_deb.txt`.
+* Python dependencies at `requirements_py3.txt`.
 
 
 Running
 -------
 
-To run these tests you need to be in the tests/acceptance directory, and the
-meta-mender-qemu layer must be next to the meta-mender layer (this layer). In
-addition you must provide a symlink, "image.dat", pointing to image you want to
-test. Normally this is the "*.ext4" image inside of your build directory, in
-"tmp/deploy/images/vexpress-qemu". Then run the tests with:
-
-```
-$ py.test
-```
-
-
-Known issues
-------------
-
-The tests currently leave behind a modified image. They won't currently pass if
-you run them again a second time without rebuilding the image first.
-
-
-Troubleshooting
----------------
-
-A common problem is that the SSH server inside the image stops working. This
-often happens if you run the tests several times after one another. It is
-unknown why, but rebuilding the Yocto image seems to make the problem go away.
+See https://github.com/mendersoftware/mender-qa/blob/master/Documentation/test-suites-overview.md#client-acceptance-tests
```

Comment:
👍 I like the centralization - less places to maintain 🙂

Reply:
Having the doc here and a pointer in `mender-qa` is also okey with me (that is the case for integ tests, for example). What is not okey is to copy it.

I guess I still have in my veins C coding principles :P